README.md

```diff
@@ -101,14 +95,39 @@ Timeline playback is multi-threaded using the C# Job System.
 Access to variant selection, model details, and payload state are all accessible via
 the inspector on the game object at which these features were authored.
 
-# Unity 2017 Users
+## Exporting USD files
+
+### Exporting via Recorder
+
+> **Prerequisite:** You need to install the [Recorder package](https://docs.unity3d.com/Packages/com.unity.recorder@latest/index.html).
+
+To export a USD composition via Recorder, you can either use the Recorder Window :
+* From the Editor main menu, select **Window > General > Recorder > Recorder Window**,
+* Then click on **+ Add Recorder**,
+* And select **USD Clip**.
+
+Or add a Recorder track :
+* From the Timeline window, right-click and select **UnityEditor.Recorder.Timeline > Recorder Track**,
+* Then right-click on the track and **add Recorder Clip**,
+* And in the **selected recorder**, choose **USD Clip**.
+
+Using Recorder is the recommended option *when compatibility with runtime is not required*.
+
+### Exporting via Usd Recorder Track (Legacy)
+
+When compatibility with runtime is required (i.e for a standalone build), the recommended option is to use the USD package Recorder Track :
+* From the Timeline window, right-click and select **Unity.Formats.USD > Usd Recorder Track**,
+* Then right-click on the track and **add USD Recorder Clip**.
+
+>  **Note:** This feature has no dependency to and is not based on the Recorder package.
+
+### Exporting via USD menu (Legacy)
+
+A last option to export a USD Composition is to use the legacy export via the Editor main menu :
+* Select **USD > Export Selected**.
 
-The official package is not compatible with 2017.4 but a separate branch is maintained to allow 2017 users to benefit from the last developments.
+Note that this option is not recommended as it offers very few export options.
```

Comment:
To be technically correct, I would change the end of the sentence to something like `as it doesn't offer any export options`.

Reply:
👍

Reply:
Done